src/simtools/simtel/pulse_shapes.py

```diff
@@ -0,0 +1,154 @@
+"""Pulse shape computations for light emission simulations for flasher."""
+
+from __future__ import annotations
```

Comment:
Is this one needed? I don't see annotations (apart from the docstring lines 68/70)